Doc/c-api/exceptions.rst

```diff
@@ -125,6 +125,41 @@ For convenience, some of these functions will always return a
 
    .. versionadded:: 3.5
 
+.. c:function:: void PyErr_ChainExceptions(PyObject *exc, PyObject *val, PyObject *tb)
+
+   Set the provided exception as the cause for the current exception. If there is no exception
```

Comment:
Does this really do cause? Where?

Reply:
There are only two exception here: the "current exception" C and the "provided exception" P.   This function sets C to be the context of P. The it makes P the new "current exception". Cause is not set.

Reply:
> Does this really do cause? Where?

That is refering to the `exception.__cause__` argument and related machinery

Reply:
I don't think this function takes a __cause__ argument. Check the code.

Reply:
> I don't think this function takes a **cause** argument. Check the code.

I am confused, unfortunately. The text says:

> Set the provided exception as the cause for the current exception. 

So the provided exception (first argument) is set as the `__cause__` for the current exception

Reply:
Ah, my bad, I am mixing things. Will clarify this sentence

Reply:
Just to clarify the confusion: I was thinking of `__context__`, not `__cause__`. The latter is for `raise ... from ...`